### PR TITLE
add: `grive2-git`

### PR DESCRIPTION
**Pull request by kwon37xi**
add grive2-git for grive2 latest version

- [x]  Edit packagelist
- [x]  Add initial pacscript
- [ ] Contact devs:  https://github.com/vitalif/grive2/issues/356 (failed to contact within 1 week)
- [x] Add maintainer to pacscript: temporarily resolved to @kwon37xi

### CODE REVIEW

File: packages/grive2-git/grive2-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name="grive2-git"
+pkgname="grive2"
+pkgdir="${STOWDIR}/${name}"
+description="Google Drive client with support for new Drive REST API and partial sync"
+url="https://github.com/vitalif/grive2.git"
+build_depends="cmake libgcrypt20-dev libyajl-dev libboost-all-dev libcurl4-openssl-dev libexpat1-dev libcppunit-dev binutils-dev debhelper zlib1g-dev dpkg-dev pkg-config"
+depends="libgcrypt20 libyajl2 libboost-all-dev libcurl4 libexpat1"
+replace="grive"
+breaks="${pkgname}"
```
Comment (wizard-28) [inline packages/grive2-git/grive2-git.pacscript:9]: Expand `breaks`

File: packages/grive2-git/grive2-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+version="$(pkgver)"
+
+prepare() {
+  mkdir build
+  cd build || exit
+}
+
+build() {
+  cmake ..
```
Comment (wizard-28) [inline packages/grive2-git/grive2-git.pacscript:23]:
```suggestion
  cmake -DCMAKE_BUILD_TYPE=Release \
        -DCMAKE_INSTALL_PREFIX=/usr ..
```

File: packages/grive2-git/grive2-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  make -j"$(nproc)"
+}
+
+install() {
+  cd build || exit
+  sudo make install DESTDIR="${pkgdir}"
+  ls -lah "${pkgdir}"
+  sleep 5
```
Comment (wizard-28) [inline packages/grive2-git/grive2-git.pacscript:31]: What's this for?

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Add `gives`

File: packages/grive2-git/grive2-git.pacscript (hunk 3 above)
Comment (kwon37xi) [reply packages/grive2-git/grive2-git.pacscript]: that was for debugging. removed.

File: packages/grive2-git/grive2-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name="grive2-git"
+pkgname="grive2"
+pkgdir="${STOWDIR}/${name}"
+description="Google Drive client with support for new Drive REST API and partial sync"
+url="https://github.com/vitalif/grive2.git"
+build_depends="cmake libgcrypt20-dev libyajl-dev libboost-all-dev libcurl4-openssl-dev libexpat1-dev libcppunit-dev binutils-dev debhelper zlib1g-dev dpkg-dev pkg-config"
+depends="libgcrypt20 libyajl2 libboost-all-dev libcurl4 libexpat1"
+replace="grive"
+breaks="grive2"
```
Comment (wizard-28) [inline packages/grive2-git/grive2-git.pacscript:9]:
```suggestion
breaks="${pkgname} ${pkgname}-bin ${pkgname}-deb ${pkgname}-app"
```

I meant this by expanding.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Now we need to wait at least 1 week for the developer to respond to your issue, if they don't within that time period we'll merge this.

File: packages/grive2-git/grive2-git.pacscript (hunk 4 above)
Comment (kwon37xi) [reply packages/grive2-git/grive2-git.pacscript]: ok, done.

File: packages/grive2-git/grive2-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,33 @@
+name="grive2-git"
+pkgname="grive2"
+pkgdir="${STOWDIR}/${name}"
+description="Google Drive client with support for new Drive REST API and partial sync"
+url="https://github.com/vitalif/grive2.git"
+build_depends="cmake libgcrypt20-dev libyajl-dev libboost-all-dev libcurl4-openssl-dev libexpat1-dev libcppunit-dev binutils-dev debhelper zlib1g-dev dpkg-dev pkg-config"
+depends="libgcrypt20 libyajl2 libboost-all-dev libcurl4 libexpat1"
+replace="grive"
+breaks="${pkgname} ${pkgname}-bin ${pkgname}-deb ${pkgname}-app"
+gives="grive2 grive"
```
Comment (wizard-28) [inline packages/grive2-git/grive2-git.pacscript:10]: Yeah... This won't work, the CI's failing to build the dummy package. @Henryws & @D-Brox  summoned.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/grive2-git/grive2-git.pacscript (hunk 5 above)
Comment (Henryws) [reply packages/grive2-git/grive2-git.pacscript:10]: Yeah you can't have two `gives`

File: packages/grive2-git/grive2-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+}
+version="$(pkgver)"
+
+prepare() {
+  mkdir build
+  cd build || exit
+}
+
+build() {
+  cmake -DCMAKE_BUILD_TYPE=Releaes \
```
Comment (Henryws) [inline packages/grive2-git/grive2-git.pacscript:24]:
```suggestion
  cmake -DCMAKE_BUILD_TYPE=Release \
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/grive2-git/grive2-git.pacscript (hunk 5 above)
Comment (kwon37xi) [reply packages/grive2-git/grive2-git.pacscript]: Actually I don't exactly understand the difference between pkgname and gives.
Anyway I set the gives value to the same name of pkgname.

**Review by wizard-28 — APPROVED** [review]
Looks good, but CI failed, so @Henryws needs to test.

**Review by Henryws — APPROVED** [review]

**Pull request merged**